TestResultSummaryService/parsers/BenchmarkMetric.js

```diff
@@ -2,7 +2,11 @@ const math = require('mathjs');
 // TODO : Find a library function for geometric mean
 function geomean(inputArray){ 
     let result = math.prod(inputArray);
+    if(inputArray.length < 2) {
+        result = math.nthRoot(result);
```

Comment:
Doesn't look right. If we just have 1 value (i.e. 100), shouldn't the geomean just be 100. With that code, won't be get 10 since the default root is 2 for `nthRoot`?